gp-entry-blocks/gpeb-edit-entry-by-unique-id.php

```
@@ -0,0 +1,201 @@
+				return array( $entry );
+			}
+		}
+
+		if ( ! $this->mapped_entry || (int) rgar( $this->mapped_entry, 'id' ) !== $entry_id ) {
+			return array();
+		}
+
+		return array( $this->mapped_entry );
+	}
+
+	public function require_matching_unique_id( $can_edit, $entry, $_current_user ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$_current_user'. (undefined)

(UnusedFormalParameter)